### ISSUE TITLE
Which configurations to use to get the results in the paper ?

### ISSUE THREAD

**Issue by AbuZekry** [kujason/avod#29]
I am trying to reproduce the results in the paper, the ones in Table 1, but I can't. I keep getting values up to 8% less than the ones in the paper. So which configuration files (for preprocessing and training) and thresholds in the evaluation script to use to reproduce the values in the paper ? Or are there any modifications to the existing configurations (preprocessing and training) to do that ?

Your help would be much appreciated, I'm working on my masters thesis :)

**Comment by AbuZekry** [comment]
No I couldn't solve it, I am using the default configuration of pyramid_cars_example.config, but the results are not like the paper at all.

**Comment by guduzxc** [comment]
You should use pyramid_cars_with_aug_example.config for the training and validation, but I only got car_detection_3D AP: 82.906799 73.472794 67.455826，which is still much lower than the paper presented, so is there any other default config should be change?

**Comment by melfm** [comment]
Try setting the `rpn_test_nms_size : 1024`, it makes the network slightly more stable. 
You could also change the learning rate schedule by setting `decay_steps:100000` and `decay_factor: 0.1`.

**Comment by guduzxc** [comment]
Thank you very much for your prompt reply. I will retrain the network with your recommendations.
Thanks for your sharing again!

**Comment by AbuZekry** [comment]
@melfm 
Should I keep the path_drop_probabilities: [0.9, 0.9], or should I set them to ones ?

**Comment by ch-lai** [comment]
@melfm How about the configuration for pedestrian and cyclist? In the paper, you got 3D AP (58.8, 49.7) in pedestrian and cyclist at IoU 0.5. However, I trained with FPN and also use augmentation "flip and pca_jitter", I only got (50.54, 44.8).

Is there any other default config should be change?
Appreciate for your help!! thanks!

**Comment by griffintin** [comment]
@melfm @AbuZekry 
I re-trained by using pyramid_cars_with_aug_example.config,  setting rpn_test_nms_size to 1024 and disabled path_drop, the detection result on validation set indeed increased a lot, small cars which failed when using pyramid_cars_example.config, now all detected. The final AP output by evaluate_object_3d_offline.cpp is even higher than that in the paper, I was wondering anything was wrong.

@kujason
 Could you share with us your configureation file for validation of 3D car detection?

**Comment by AbuZekry** [comment]
@hulaifeng I didn't experiment on the test set. I am working the training/validation split

**Comment by AbuZekry** [comment]
@hulaifeng I didn't experiment on the test set. I am working the training/validation split

**Comment by tilltheworldend** [comment]
@guduzxc Hi.I ran the codes with pyramid_cars_with_aug_example configuration and didn't change any parameters in it.But I only got （76.8，72.4，66.9）ap using the 120000 checkpoint.Do you know why cause this?

**Comment by Hub-Tian** [comment]
@tilltheworldend I also confront with the same problem, and I find that if you train the model for several times, the results vary greatly. Does anyone know why?

**Comment by csjxchen** [comment]
> @tilltheworldend I also confront with the same problem, and I find that if you train the model for several times, the results vary greatly. Does anyone know why?

I got the same problem.Have you solved it?

**Comment by csjxchen** [comment]
Who knows which config file corresponds to the base Network and which config corresponds to the Feature Pyramid Extractor model in TABLE 3?

**Comment by kujason** [comment]
See https://github.com/kujason/avod/issues/10#issuecomment-373937017
